cmd/snap-repair/runner.go

```diff
@@ -80,6 +80,21 @@ func (r *Repair) SetStatus(status RepairStatus) {
 	r.run.SaveState()
 }
 
+//
+func makeRepairSymlink() (tmpdir string, err error) {
+	// make "repair" binary available to the reapir scripts via symlink
+	// to the real snap-repair
+	dir, err := ioutil.TempDir("", "repair-")
```

Comment:
wondering if it isn't simpler to just always create the same symlink (or ignore if it's already there) in "/run/snapd/repair/tools/repair"